auto_py_to_exe/web/js/i18n.js

```diff
@@ -20,6 +21,7 @@ const translationMap = {
                 pt_br: 'Arquivo Único',
                 sp: 'Archivo Único',
                 tr: 'Tek Dosya',
+                th:'หนึ่งไฟล์',

```

Comment:
To save me some time fixing these, can we put a space between `:` and `'`, e.g. ` th: 'หนึ่งไฟล์',`